packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1541,7 +1538,12 @@ class StripeHelper {
       invoiceTotalCurrency,
       cardType,
       lastFour,
-      nextInvoiceDate: new Date(nextInvoiceDate * 1000),
+      // TODO: According to Stripe, this value will be null for invoices where collection_method=send_invoice
```

Comment:
This seems to work, but I *am* scratching my head at whether we'll hit this case where nextInvoiceDate is null on reactivating a cancelled subscription. Doesn't seem like any of our subscriptions should be collection_method=send_invoice, even if they're trial subscriptions?